index.md

```diff
@@ -5,10 +5,13 @@ version: develop
 latest: true
 ---
 
-This pipeline can be used to process germline-DNA data, starting from FastQ
+This pipeline can be used to process germline-DNA data, starting with FastQ
 files. It will perform adapter clipping (using cutadapt), mapping (using BWA
 mem) and variantcalling (based on the GATK Best Practises).
 
+This pipeline is part of [BioWDL](https://biowdl.github.io/)
+developed by [the SASC team](http://sasc.lumc.nl/).
+
 ## Usage
 In order to run the complete multisample pipeline, you can
 run `pipeline.wdl` using
```

Comment:
I would remove "In order to run the complete multisample pipeline, you can run `pipeline.wdl` using" since it doesn't give much additional value and multisample could lead to unnecessary confusion.

I would add a line like "Cromwell program <link to cromwell website> is needed to run this pipeline. It can be installed manually or executed from a loaded environment module (e.g., on LUMC Shark cluster for LUMC local users).